cmd/helm/flags.go

```diff
@@ -47,6 +47,7 @@ func addChartPathOptionsFlags(f *pflag.FlagSet, c *action.ChartPathOptions) {
 	f.StringVar(&c.CertFile, "cert-file", "", "identify HTTPS client using this SSL certificate file")
 	f.StringVar(&c.KeyFile, "key-file", "", "identify HTTPS client using this SSL key file")
 	f.StringVar(&c.CaFile, "ca-file", "", "verify certificates of HTTPS-enabled servers using this CA bundle")
+	f.StringVar(&c.Renegotiate, "renegotiate", "", "TLS renegotiation strategy, valid options include Never(default), OnceAsClient and FreelyAsClient")
```

Comment:
the default here appears to be the empty string. should it be "never" instead, similar to repo_add.go?

Reply:
Yes, done